### PR TITLE
[TMVA][SOFIE] Add support for Conv Operator in Keras parser

### PR DESCRIPTION
**Pull request by sanjibansg**
This PR adds support for parsing Convolution layers from PyTorch and Keras models into an RModel object.

## Progress
- [x] Support for Keras Convolution Layers(Conv2D)          
  - [x] Support for `padding="valid"`
  - [x] Support for `padding="same"`
  - [x]  Tests

## Checklist:
- [x] tested changes locally

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default, ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2204/default, ROOT-ubuntu2004/default, ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On

**Review by lmoneta — APPROVED** [review]
LGTM!
Thank you for this useful addition!
